src/main/java/service/TransactionParser.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public interface TransactionParser {
```

Comment:
For structure use more general names
```suggestion
public interface Parser {
```